### ISSUE TITLE
All Spanish speakers are generic Latin American

### ISSUE THREAD

**Issue by linguax-gc** [suno-ai/bark#408]
I hope future versions will include different Spanish accents, at least for European Spanish, but possible for other varieties (Rioplatense, Highlands, Central American, Caribbean, Mexican).

Amazing work! Thank you so much for doing this!

**Comment by EricKong1985** [comment]
how to  fix the accent issue? Looks except English, all other language have accent issue!

**Comment by juangea** [comment]
I agree, right now all spanish speakers are Latin American, is there a way to train or modify some speaker to have a Spanish accent?
